### ISSUE TITLE
retesteth alethIPCDebug: don't hardcode absolute path to the socket

### ISSUE THREAD

**Issue by gumb0** [ethereum/tests#653]
https://github.com/ethereum/tests/blob/59490b4fbcfc8cf2279029146193b252a2432b7e/Retesteth/alethIPCDebug/config#L4

This works only for wins.

**Comment by winsvega** [comment]
You supposed to change it to your local path.

**Comment by gumb0** [comment]
Perhaps this should be a command line argument to retesteth instead of config file. With a default being `~/.ethereum/geth.ipc`

**Comment by gumb0** [comment]
On the other hand, if user is supposed to edit these files locally often, I think it shouldn't happen inside the git repo. retesteth should copy them on the first run into some dir like maybe `~/.retesteth` and then read them from there always.

**Comment by gumb0** [comment]
I don't really understand why these configs live here in tests repo, either. They're very retesteth-specific, it seems they should be there.

**Comment by gumb0** [comment]
It's very annoying to deal with local changes in git repo, when you want to update tests / change tests branch etc.

Also there's no reason to tie changes in config formats to the changes in tests - say I want to try various versions of tests, switching between the branches. But if you also make changes to what is the correct format of configs, only some branches will work, only those where configs have the format corresponding to my retesteth version.

In other words, it would be better if the code parsing the configs is changed in sync with changes in configs themselves, would be simpler if this happens in a single repo (retesteth)

**Comment by winsvega** [comment]
perhaps move it to the retesteth repo?

you are not supposed to change this configs often actually.

**Comment by gumb0** [comment]
retesteth repo would be a better place, I think

**Comment by winsvega** [comment]
Then it looks like if to move this configs to retesteth repo it will integrate it on compilation time and then a binary deploy to . retesteth config folder

**Comment by gumb0** [comment]
Sounds good to me.

**Comment by winsvega** [comment]
now retesteth issue
